### PR TITLE
DAOSGCP-75 Example and documentation updates

### PR DESCRIPTION
**Pull request by markaolson**
1. Renamed example directories for clarity and consistency
2. Updated variable names in examples to be consistent
3. Added terraform.tfvars.*.example files.
   Values in the files will need to be updated in a following commit
4. Updated main README.md to contain more instructions for getting started
5. Created Cloud Shell tutorial that walks through the one-time setup steps
6. Created Cloud Shell tutorial that walks through the daos_cluster example

### CODE REVIEW

File: terraform/examples/only_daos_server/terraform.tfvars.perf.example [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+server_preemptible         = "false"
+server_mig_name            = "daos-server"
+server_template_name       = "daos-server"
+server_instance_base_name  = "daos-server"
+server_machine_type        = "n2-custom-36-262144"
+server_os_family           = "daos-server-centos-7"
+server_os_project          = "<project_id>"
+server_os_disk_type        = "pd-ssd"
+server_os_disk_size_gb     = 20
```
Comment (cboneti) [inline terraform/examples/only_daos_server/terraform.tfvars.perf.example:19]: Are these examples inverted? this looks like the TCO setup whereas the other one looks like perf.

**Review by cboneti — COMMENTED** [review]
Thank you for this. I think it greatly improves the overall quality of this repo and makes this much more usable. 
My only comment is that the tvars examples seem to be inverted...

File: terraform/examples/only_daos_server/terraform.tfvars.perf.example (hunk 1 above)
Comment (Kmannth) [reply terraform/examples/only_daos_server/terraform.tfvars.perf.example:19]: We fixed this on the call today.

**Review by Kmannth — DISMISSED** [review]
Looks great Mark.   Thanks for helping with the 2 T-Shirt Sizes.

**Review by cboneti — DISMISSED** [review]

**Review by cboneti — APPROVED** [review]

**Review by Kmannth — APPROVED** [review]

**Pull request merged**